app/views/tag/blog.html.erb

```diff
@@ -1,36 +1,35 @@
 <div class="col-md-2">
-
 </div>
 <div class="col-md-8 blog">
   <div class="blog-header">
-    <h1><%= params[:id] == "blog" ? "The Public Lab" : params[:id].split('-').join(' ').capitalize %> Blog</h1>
+    <h1><%= params[:id] == "blog" ? "The Public Lab" : params[:id].split('-').join(' ').capitalize %> <%= t('tag.blog.blog') %></h1>
 
     <% if params[:id] == "blog" %>
       <br />
-      <p>stories from the Public Lab community</p>
-      <p><a href="/wiki/blog">About the blog</a></p>
+      <p><%= t('tag.blog.stories_from_community') %></p>
+      <p><a href="/wiki/blog"><%= t('tag.blog.about_the_blog') %></a></p>
     <% end %>
 
     <br />
  
     <% if current_user %>
-      <a class="btn btn-sm" href="/feed/tag/<%= params[:id] %>.rss"><i class="fa fa-rss"></i> RSS</a>
+      <a class="btn btn-sm" href="/feed/tag/<%= params[:id] %>.rss"><i class="fa fa-rss"></i> <%= t('tag.blog.rss') %></a>
```

Comment:
RSS is a standard; is it typically translated, perhaps for different character sets? Or are we OK leaving it as "RSS"?